ios-base/DroidKaigi 2020/ApplicationScheme.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ApplicationScheme.swift
+//  DroidKaigi 2020
+//
+//  Created by 伊藤凌也 on 2020/01/17.
```

Comment:
I'm very grateful, but this repository does not currently include the author's name in source code comments. 🙇 
But If it is difficult to remove, this is fine.

Reply:
ok. I will do! 👍 
change files are not so much, so it's not burdensome!